ui/src/components/policy/PolicyRuleTable.js

```diff
@@ -182,6 +182,7 @@ export default class PolicyRuleTable extends React.Component {
                 this.props.domain,
                 this.props.name,
                 this.state.deleteAssertionId,
+                'deleted using Athenz UI',
```

Comment:
store this as a const and use it anywhere we want to pass it as audit reference

Reply:
done